src/prng/mod.rs

```diff
@@ -145,6 +145,21 @@
 //! unpredictable. This implies there must be no obvious correlations between
 //! output values.
 //!
+//! ### Quality stars:
+// 5. reserved for crypto-level (e.g. ChaCha8, ISAAC)
```

Comment:
Good work on the stars!

You need to fix the `//!` at the start

Reply:
I wanted to document *some* reasoning (otherwise it seems too subjective), but thought the exact details not worthy enough for the documentation.

Reply:
I see. Not really sure whether this is a good idea, but it'll do for now.